opentech/apply/funds/models.py

```diff
@@ -219,9 +244,29 @@ class Round(SubmittableStreamForm):
                 FieldPanel('start_date'),
                 FieldPanel('end_date'),
             ]),
-        ], heading="Dates")
+        ], heading="Dates"),
+        ReadOnlyPanel('workflow'),
+        ReadOnlyInlinePanel('forms'),
```

Comment:
Unfortunately this does not indicate what form is used when creating the round.

Reply:
Yeah, It is a tricky one. Perhaps some help text to describe what is happening

Reply:
Text added to improve clarity around form behaviour